stable/aws-service-events-exporter/values.yaml

```diff
@@ -5,10 +5,10 @@
 replicas: 1
 
 image:
-  repository: thomasnyambati/aws-service-events-exporter
```

Comment:
You should leave this as is since the chart is public. change the image with values instead.

Reply:
makes sense, will change it.